crates/csharp/src/csproj.rs

```diff
@@ -161,23 +162,20 @@ impl CSProjectMonoBuilder {
         let camel = format!("{}World", world.to_upper_camel_case());
 
         let aot = self.aot;
-
-        let maybe_aot = match aot {
-            true => format!("<WasmBuildNative>{aot}</WasmBuildNative>"),
-            false => String::new(),
-        };
+        let tfm = "net9.0";
```

Comment:
Should we move to Net10 ? 
Because we are not going to backport fixes to Net9, right ?
One more below.

Reply:
there won't be a working net10 sdk for a while unfortunately